tikv/kv.go

```diff
@@ -584,6 +584,23 @@ func (s *KVStore) updateSafeTS(ctx context.Context) {
 	tikvClient := s.GetTiKVClient()
 	wg := &sync.WaitGroup{}
 	wg.Add(len(stores))
+	// Try to get the minimum resolved timestamp of the store from PD.
+	var (
+		storeIDs []string
+		err      error
+	)
+	for _, store := range stores {
+		storeIDs = append(storeIDs, strconv.FormatUint(store.StoreID(), 10))
+	}
+	StoreMinResolvedTSs := make(map[uint64]uint64)
```

Comment:
Why make it exported?

Reply:
for `for` block will use it.

Reply:
The exportable attribute is only meaningful when referenced across packages. The definition and use of `StoreMinResolvedTSs` are in the same function, so it is meaningless to make it exportable here.

Reply:
Sorry... I misunderstand your mean...